spec/MongoStorageAdapter.spec.js

```diff
@@ -286,4 +286,29 @@ describe_only_db('mongo')('MongoStorageAdapter', () => {
       done();
     });
   });
+
+  it('getClass if exists', async () => {
+    const adapter = new MongoStorageAdapter({ uri: databaseURI });
+
+    const schema = {
+      fields: {
+        array: { type: 'Array' },
+        object: { type: 'Object' },
+        date: { type: 'Date' },
+      },
+    };
+
+    await adapter.createClass('MyClass', schema);
+    const myClassSchema = await adapter.getClass('MyClass');
+    expect(myClassSchema).toBeDefined();
+  });
+
+  it('getClass if not exists', async () => {
+    const adapter = new MongoStorageAdapter({ uri: databaseURI });
+    try {
+      await adapter.getClass('UnknownClass');
+    } catch (e) {
+      expect(e).toBeUndefined();
```

Comment:
does this throw or not?  i.e. is the test at 311 tested?  and if so why would e be undefined?

Reply:
Because of this PR `getClass` is no longer used anywhere. I didn't want to remove it or should I?

it `throws undefined`, I don't know the reason behind it.